modules/gapi/src/backends/streaming/gstreamingbackend.cpp

```diff
@@ -0,0 +1,211 @@
+//
+// Copyright 2020 Intel Corporation.
+//
+// This software and the related documents are Intel copyrighted materials,
+// and your use of them is governed by the express license under which they
+// were provided to you ("License"). Unless the License provides otherwise,
+// you may not use, modify, copy, publish, distribute, disclose or transmit
+// this software or the related documents without Intel's prior written
+// permission.
+//
+// This software and the related documents are provided as is, with no
+// express or implied warranties, other than those that are expressly
+// stated in the License.
+//
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/gapi/util/throw.hpp> // throw_error
+#include <opencv2/gapi/streaming/streaming.hpp> // kernels
+
+#include "api/gbackend_priv.hpp"
+#include "backends/common/gbackend.hpp"
+
+#include "gstreamingbackend.hpp"
+#include "gstreamingkernel.hpp"
+
+namespace {
+
+struct StreamingCreateFunction
+{
+    static const char *name() { return "StreamingCreateFunction";  }
+    cv::gapi::streaming::CreateActorFunction createActorFunction;
+};
+
+using StreamingGraph = ade::TypedGraph
+    <
+      cv::gimpl::Op
+    , StreamingCreateFunction
+    >;
+
+using ConstStreamingGraph = ade::ConstTypedGraph
+    <
+      cv::gimpl::Op
+    , StreamingCreateFunction
+    >;
+
+
+class GStreamingExecutable final: public cv::gimpl::GIslandExecutable
```

Comment:
The name is probably not very good because not all streaming operations work through this backend
What do you think ?

Reply:
Since these operations are `intrinsics` to us, I'd call it `GStreamingIntrinExecutable` and so on

Reply:
Renamed this one to `GStreamingIntrinExecutable`. Are there other entities to rename?